modules/porous_flow/src/actions/PorousFlowActionBase.C

```diff
@@ -66,6 +74,15 @@ PorousFlowActionBase::PorousFlowActionBase(const InputParameters & params)
 void
 PorousFlowActionBase::act()
 {
+  const auto & all_subdomains = _problem->mesh().meshSubdomains();
+  if (all_subdomains.empty())
+    mooseError("No subdomains found");
+  _coord_system = _problem->getCoordSystem(*all_subdomains.begin());
+  for (auto subdomain : all_subdomains)
```

Comment:
You should probably get in the habit of using an `auto &` in range-for loops. You don't need it for something like this where the type is a POD, but it matters when you are looping over a more complex type. It's just good to be consistent.

Reply:
fixed